### PR TITLE
Adds FendCalculator

### PR DESCRIPTION
**Pull request by IsaacTay**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+        "Name": "FendCalculator",
+        "Description": "Arbitrary-precision unit-aware calculator. https://printfn.github.io/fend/",
+        "Author": "IsaacTay",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/IsaacTay/Flow.Launcher.Plugin.FendCalculator",
+        "UrlDownload": "https://github.com/IsaacTay/Flow.Launcher.Plugin.FendCalculator/releases/download/v1.0.0/Flow.Launcher.Plugin.FendCalculator.zip",
+        "UrlSourceCode": "https://github.com/IsaacTay/Flow.Launcher.Plugin.FendCalculator/tree/main",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/IsaacTay/Flow.Launcher.Plugin.FendCalculator@main/Images/calculator.png",
+    }
```
Comment (Garulf) [inline plugins.json:13]:
```suggestion
    },
```

File: plugins.json [hunk 2]
```diff
@@ -1,4 +1,16 @@
 [
+    {
+        "ID": "E5E5A4BD-39C7-4B41-93F8-0D4850BC317C",
+        "Name": "FendCalculator",
+        "Description": "Arbitrary-precision unit-aware calculator. https://printfn.github.io/fend/",
+        "Author": "IsaacTay",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/IsaacTay/Flow.Launcher.Plugin.FendCalculator",
+        "UrlDownload": "https://github.com/IsaacTay/Flow.Launcher.Plugin.FendCalculator/releases/download/v1.0.0/Flow.Launcher.Plugin.FendCalculator.zip",
```
Comment (jjw24) [inline plugins.json:10]:
```suggestion
        "UrlDownload": "https://github.com/IsaacTay/Flow.Launcher.Plugin.FendCalculator/releases/download/v1.0.1/Flow.Launcher.Plugin.FendCalculator.zip",
```

**Review by jjw24 — APPROVED** [review]
I would recommend adding a function to prompt user to select their Fend Calculator installation path or install it automatically. To install it automatically for users, you can use [Droplex](https://github.com/jjw24/Droplex), see this Putty plugin's [usage](https://github.com/jjw24/Flow.Launcher.Plugin.Putty/blob/f1c32489dcb24f1f1d3c933727677d6da4d284b9/Flow.Launcher.Plugin.Putty/Plugin.cs#L107) as an example.

Perhaps also mention in the readme that Fend Calculator need to be installed before using the plugin.

**Comment by IsaacTay** [comment]
Will do, thanks for the feedback!

**Pull request merged**